lua/autorun/photon/cl_emv_meta.lua

```diff
@@ -1006,4 +1006,385 @@ function EMVU:MakeEMV( emv, name )
+						local distBehind = v:WorldToLocal(ply:GetPos())[2]
+						
+						if IsValid(ply:GetVehicle()) then
+							if ply:GetVehicle():GetParent() == v then
+								if v.Siren:GetVolume() ~= thirdPersonVolume and ply:GetVehicle():GetThirdPersonMode() then
+									v.Siren:ChangeVolume(thirdPersonVolume, updateRate)
+								elseif v.Siren:GetVolume() ~= interiorVolume and !ply:GetVehicle():GetThirdPersonMode() then
+									v.Siren:ChangeVolume(interiorVolume, updateRate)
+								end
+							end
+						else
+							if v.Siren:GetVolume() ~= 1 and distBehind > 0 then
+								v.Siren:ChangeVolume(1, updateRate)
+							elseif distBehind < 0 then
+								v.Siren:ChangeVolume(math.Clamp(1 + (distBehind/fadeDist), 0.05, 1), updateRate)
+							end
+						end
+
+						if math.abs(doppler) > 1 then
+								v.Siren:ChangePitch(math.Clamp(100 + doppler, 50, 150), updateRate)
+							elseif v.Siren:GetPitch() ~= 100 then
+								v.Siren:ChangePitch(100, updateRate)
+						end
+					else
+						if v.Siren:GetPitch() ~= 100 then
+							v.Siren:ChangePitch(100)
+						end
+						if v.Siren:GetVolume() ~= thirdPersonVolume and v:GetThirdPersonMode() then
+							v.Siren:ChangeVolume(thirdPersonVolume, updateRate)
+						elseif v.Siren:GetVolume() ~= interiorVolume and !v:GetThirdPersonMode() then
+							v.Siren:ChangeVolume(interiorVolume, updateRate)
+						end
+					end
+				end
+
+				if v.Siren2 then
+					if (IsValid(v:GetDriver()) and v:GetDriver() ~= ply) or !IsValid(v:GetDriver()) then
+						local distBehind = v:WorldToLocal(ply:GetPos())[2]
+						
+						if IsValid(ply:GetVehicle()) then
+							if ply:GetVehicle():GetParent() == v then
+								if v.Siren2:GetVolume() ~= thirdPersonVolume and ply:GetVehicle():GetThirdPersonMode() then
+									v.Siren2:ChangeVolume(thirdPersonVolume, updateRate)
+								elseif v.Siren2:GetVolume() ~= interiorVolume and !ply:GetVehicle():GetThirdPersonMode() then
+									v.Siren2:ChangeVolume(interiorVolume, updateRate)
+								end
+							end
+						else
+							if v.Siren2:GetVolume() ~= 1 and distBehind > 0 then
+								v.Siren2:ChangeVolume(1, updateRate)
+							elseif distBehind < 0 then
+								v.Siren2:ChangeVolume(math.Clamp(1 + (distBehind/fadeDist), 0.05, 1), updateRate)
+							end
+						end
+
+						if math.abs(doppler) > 1 then
+								v.Siren2:ChangePitch(math.Clamp(100 + doppler, 50, 150), updateRate)
+							elseif v.Siren2:GetPitch() ~= 100 then
+								v.Siren2:ChangePitch(100, updateRate)
+						end
+					else
+						if v.Siren2:GetPitch() ~= 100 then
+							v.Siren2:ChangePitch(100)
+						end
+						if v.Siren2:GetVolume() ~= thirdPersonVolume and v:GetThirdPersonMode() then
+							v.Siren2:ChangeVolume(thirdPersonVolume, updateRate)
+						elseif v.Siren2:GetVolume() ~= interiorVolume and !v:GetThirdPersonMode() then
+							v.Siren2:ChangeVolume(interiorVolume, updateRate)
+						end
+					end
+				end
```

Comment:
Do you think there's some way to loop these, so we're not having to rewrite the same code?